SharedSource/Core/RoslynAnnotations/NullableAnnotations.cs

```diff
@@ -1,5 +1,4 @@
-﻿﻿
-#define INTERNAL_NULLABLE_ATTRIBUTES
+﻿#define INTERNAL_NULLABLE_ATTRIBUTES
 // https://github.com/dotnet/corefx/blob/48363ac826ccf66fbe31a5dcb1dc2aab9a7dd768/src/Common/src/CoreLib/System/Diagnostics/CodeAnalysis/NullableAttributes.cs

```

Comment:
need a matching LM file and "correct" / complete license header. compare with ReSharper annotations.